code/lib/preview/src/modules/addons.ts

```diff
@@ -0,0 +1,2 @@
+/* eslint-disable import/no-extraneous-dependencies */
+export * from '@storybook/addons';
```

Comment:
Do we still need all these re-exports?

Reply:
yes? this is where the implementation comes from..

Reply:
Couldn't we change `runtime.ts` from:

```ts
import * as ADDONS from '../modules/addons';
```

to 

```ts
import * as ADDONS from '@storybook/addons';
```